x/ecocredit/base/keeper/query_params_test.go

```diff
@@ -55,6 +55,9 @@ func TestQuery_Params(t *testing.T) {
 	})
 	assert.NilError(t, err)
 
+	allowedChain := "polygon"
+	assert.NilError(t, s.stateStore.AllowedBridgeChainsTable().Insert(s.ctx, &api.AllowedBridgeChains{ChainName: allowedChain}))
```

Comment:
A bit confused why in some cases we are using all caps and some cases lowercase. I would prefer to stick with lower case given the bridge service is expecting `polygon` at the moment.

Reply:
i can update this to upper (depending on the choice for case). 

bridge service can call with whatever text-case it wants, each `AllowedBridgeChains` table functions all convert the request to upper case before reading/writing

Reply:
I would prefer all lowercase but maybe not necessary.